packages/frontend/package.json

```diff
@@ -59,6 +57,7 @@
     "postcss": "^8.4.14",
     "prettier": "^2.8.4",
     "prettier-plugin-tailwindcss": "^0.2.2",
-    "tailwindcss": "^3.0.24"
+    "@tailwindcss/container-queries": "^0.1.1",
```

Comment:
Really cool that this is so widely supported https://caniuse.com/css-container-queries. Didn't even know this existed!